src/style.css

```diff
@@ -0,0 +1,33 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--1 div:nth-child(-n + 1),
+.stars--2 div:nth-child(-n + 2),
+.stars--3 div:nth-child(-n + 3),
+.stars--4 div:nth-child(-n + 4),
+.stars--5 div:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--0 div:nth-child(-n + 4),
+.stars--1 div:nth-child(-n + 4),
+.stars--2 div:nth-child(-n + 4),
+.stars--3 div:nth-child(-n + 4),
+.stars--4 div:nth-child(-n + 4),
+.stars--5 div:nth-child(-n + 4) {
+  margin-right: 4px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
  background-image: url(images/star-active.svg);
}

.stars__star {
  height: 16px;
  width: 16px;
  background-image: url(images/star.svg);
  background-repeat: no-repeat;
  background-position: center;
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```
style only by classes. by tags maybe can be only html, body,  reset some default styles.

Reply:
https://mate.academy/learn/css-basics/pseudo-elements-and-classes?section=video&videoId=845
In this video-example used tags "<li>". Is it incorrect video?

next, green example of code doesn't work in "npm test"
![image](https://user-images.githubusercontent.com/126285904/228512009-06b41bfc-5a21-42e2-b8a8-726e1d64c24f.png)

Reply:
Works.